tests/unit/lib/kwlib_test.sh

```diff
@@ -965,4 +965,25 @@ function test_show_verbose()
   assert_equals_helper 'Expected value of command' "$LINENO" "$cmd" "$output"
 }
 
+function test_create_shared_memory_dir()
+{
+  local output
+
+  TMP_DIR="${SHUNIT_TMPDIR}/test/tmp_dir"
+  mkdir --parents "$TMP_DIR"
+
+  output=$(create_shared_memory_dir)
+  assertContains "Unable to create TMP_DIR in the specified directory" "$output" "$TMP_DIR"
```

Comment:
Use single-quotes in the assert message.

Reply:
Tbh... I'm not sure if I liked this `prefix` test... it is ok, but not deep enough in terms of validation. How about checking if the `TMP_DIR` folder is clean; next you run the `create_shared_memory_dir`, and you should expect a new folder to become part of the `TMP_DIR`. You can even collect the return of the function, and use it to see if it exists in the assert (e.g., `assertTrue ... [[ -d "$output" ]]`.

Reply:
> Use single-quotes in the assert message.

Just to be shure, even in the assert messages from the shunit2 should we use single quotes? because all the assertEquals, even the ones without variable expansion, uses double quotes like in 'test_find_kernel_root()' or in 'test_join_path()'

Reply:
Hey João. I see how this may cause some confusion. There are some parts of kw that are indeed outdated, so don't take them as reference. The general rule is if there are no expansions, use single quotes.